docs/documentation/deployment.md

```diff
@@ -35,6 +35,12 @@ server.run(PORT, () => {
 });
 ``` 
 
+Install `koa-static` and `koa-mount`
+
+```
+npm i koa-static koa-mount
```

Comment:
Echoing what @delucis mentioned, let's change this to `npm i --save koa-static koa-mount`.

Thanks for the Deployment section.  Looks great!